setup/mysql.sql

```diff
@@ -145,10 +147,25 @@ INSERT INTO `vexim`.`users`
     smtp, pop, realname, type, admin
 )
 VALUES 
-(   '1', 'siteadmin', 'siteadmin', '$1$12345678$2lQK5REWxaFyGz.p/dos3/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
+(   '1', 'siteadmin', 'siteadmin', '$6$uR.EiB1dj5rrvwMF$Qh5LgdjOZavKXwhi9IF0Yuzu7qxsG.dLTTB8e./55ZRNfBuZVLnfUSOEXa0oWT6174myO.WYkOy83HYWAKPbK/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
```

Comment:
Cyrus: Uses pam_mysql, good question if this works with sha512.
mysql-popper: uses crypt() of the system
tpop3d: ?
qpop-mysql: ?

Debian Jessie does use SHA512 in /etc/shadow, so I suppose that crypt() and pam_mysql should be possible.

But this does not solve our problem. Even if we set "CHANGE" in cleartext, enforced a password change after the first login (then with the selected hashing scheme), this wouldn't enable any of the POP/IMAP server to be able to speak our crypt scheme. We need to clarify before which crypt-schemes our IMAP/POP-server (and exim in case it isn't courier/dovecot) can handle.